client/state/sites/updates/test/selectors.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isRequestingSiteUpdates,
+	getUpdatesBySiteId
+} from '../selectors';
+
+describe( 'selectors', () => {
+	describe( '#isRequestingSiteUpdates()', () => {
+		it( 'should return false if site updates have not been fetched yet', () => {
+			const isRequesting = isRequestingSiteUpdates( {
+				sites: {
+					updates: {
+						requesting: {}
+					}
+				}
+			}, 12345678 );
+
+			expect( isRequesting ).to.be.false;
+		} );
+
+		it( 'should return false if the site updates are not being requested', () => {
+			const isRequesting = isRequestingSiteUpdates( {
+				sites: {
+					updates: {
+						requesting: {
+							12345678: false
+						}
+					}
+				}
+			}, 12345678 );
+
+			expect( isRequesting ).to.be.false;
+		} );
+
+		it( 'should return true if the site updates are being requested', () => {
+			const isRequesting = isRequestingSiteUpdates( {
+				sites: {
+					updates: {
+						requesting: {
+							12345678: true
+						}
+					}
+				}
+			}, 12345678 );
+
+			expect( isRequesting ).to.be.true;
+		} );
+	} );
+
+	describe( '#getUpdatesBySiteId()', () => {
+		it( 'should return an empty array if site updates have not been fetched yet', () => {
+			const updates = getUpdatesBySiteId( {
+				sites: {
+					updates: {
+						items: {}
+					}
+				}
+			}, 12345678 );
+
+			expect( updates ).to.eql( [] );
+		} );
+
+		it( 'should return the updates for an existing site', () => {
+			const exampleUpdates = {
+				plugins: 1,
+				total: 1,
+			};
```

Comment:
Hm, not, the response format seems fine. So maybe the "empty" case should use an empty object instead of an array?